src/olympia/addons/tests/test_views.py

```diff
@@ -927,6 +928,7 @@ def test_queries(self):
             # - 1 files
             # - 1 licenses
             # - 1 licenses translations
+            # - 2 queries for webext_permissions - FIXME - there should only be 1
```

Comment:
this is a test that found one of the unperformant queries i mentioned.  There will be an issue filed because I don't think this pr needs to be any bigger!